schemas/latest/landingzones/types/appServiceLinuxContainer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "definitions": {
+      "APPSERVICELINUXCONTAINER": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "value": {
+            "$ref": "#/definitions/Value"
+          }
+        },
+        "required": [
+          "value"
+        ],
+        "title": "AppServiceLinuxContainer"
+      },
+      "Value": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "enabled": {
+            "type": "boolean",
+            "enum": [
```

Comment:
If enabled = false, we should not expect `skuName` and `skuTier` parameters since they have no purpose.

Reply:
Thanks Senthuran for fixing this!